pom.xml

```diff
@@ -4,9 +4,9 @@
   <modelVersion>4.0.0</modelVersion>
 
   <properties>
-    <version>2.0.0</version>
-    <maven-version>${version}-DEV-UNOFFICIAL</maven-version>
-    <current-date>${maven.build.timestamp}</current-date>
+    <betonquest.version>2.0.0</betonquest.version>
+    <betonquest.version.qualifiers>${betonquest.version}-DEV-UNOFFICIAL</betonquest.version.qualifiers>
```

Comment:
I think it should be `betonquest.version.qualifier` not `betonquest.version.qualifiers`